test/embedding.jl

```diff
@@ -0,0 +1,12 @@
+# test that the embedding example runs without error
+let
+    if is_windows()
+        embedding_bin = joinpath(JULIA_HOME,"embedding.exe")
+    else
+        embedding_bin = joinpath(JULIA_HOME,"..","libexec","embedding")
```

Comment:
maybe, would have to read through the past discussions to remind myself why libexec was being used.

could save a relative path constant populated from the makefile like DATAROOTDIR if we keep it here